modules/Db2/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "db2_host_address" {
```

Comment:
The CP4BA module needs these outputs:   
DB2_PORT_NUMBER         = var.db2_port_number
DB2_HOST_NAME           = var.db2_host_name
DB2_HOST_IP             = var.db2_host_ip

What does the `db2_host_address` refer to?  DB2_HOST_NAME   or DB2_HOST_IP?

Also, you need to make sure that https://github.com/ibm-hcbt/cloud-pak-sandboxes/blob/main/terraform/cp4ba/main.tf grabs these outputs from the db2 module, not user-defined variables.

Reply:
`db2_host_address` is referring to DB2_HOST_NAME. 
`joel-local-cp4aiops-vpc-c-c0b572361ba41c9eef42d4d51297b04b-0000.us-south.containers.appdomain.cloud`